src/EUTelGeometryTelescopeGeoDescription.cpp

```diff
@@ -737,6 +738,35 @@ Eigen::Matrix3d EUTelGeometryTelescopeGeoDescription::getFlipMatrix(int sensorID
 	return flipMat;
 }
 
+int EUTelGeometryTelescopeGeoDescription::getSensorIDFromManager()  {
```

Comment:
This is used to get the sensorID of the node which the geomanager is pointing to. I need it in my radiation length calculations. Since I need to be able to identify which nodes are which.